README.md

```diff
@@ -3,202 +3,212 @@
 [ci-badge]: https://travis-ci.org/firebase/firebase-jobdispatcher-android.svg?branch=master
 [ci-link]: https://travis-ci.org/firebase/firebase-jobdispatcher-android
 
-The Firebase Android JobDispatcher is a library that provides a high-level wrapper around job
-scheduling engines on Android, starting with the [GCM Network Manager][nts].
+The Firebase JobDispatcher is a library for working with the [JobScheduler][] compatibility library
+in Google Play services. This replaces the old [GCM Network Manager][nts] library.
 
-This replaces the old GCM Network Manager library.
+## Overview
 
-## Installation
+### What's a [JobScheduler][]?
 
-For now, clone the repo:
-```
-git clone https://github.com/firebase/firebase-jobdispatcher-android
-cd firebase-jobdispatcher-android
-```
-
-Build the `aar` bundle:
-```
-./gradlew aar
-```
-And copy it to where you need it:
-```
-cp jobdispatcher/build/outputs/aar/jobdispatcher-release.aar
-```
-
-## Concepts
-
-### Job
-
-A Job is a description of a unit of work. At its heart, it's composed of a
-series of mandatory attributes:
+The JobScheduler is an Android system service available on API levels 21 (Lollipop)+. It provides an
+API for scheduling units of work (represented by [`JobService`][JobService] subclasses) that will be
+executed in your app's process.
 
--   A string tag that (within your app) uniquely identifies the job.
+### Why is this better than whatever I'm doing now?
```

Comment:
s / whatever I'm doing now / background services and listening for system broadcast